include/nbl/ui/CWindowX11.h

```diff
@@ -37,11 +38,18 @@ class CWindowX11 final : public IWindowX11
 		return core::smart_refctd_ptr<CWindowX11>(win, core::dont_grab);
 	}
 
+	void processEvent(XEvent event);
 private:
     CWindowX11(core::smart_refctd_ptr<system::ISystem>&& sys, uint32_t _w, uint32_t _h, E_CREATE_FLAGS _flags);
 
     Display* m_dpy;
     native_handle_t m_native;
+	CWindowManagerX11* m_manager;
+	// Not sure yet if i need these or i can do without them 
+	bool isMaximized = false, isMinimized = false; 
```

Comment:
`IWindow::m_flags` should be always updated to tell you whether the window is minimized, maximized or neither

Reply:
so those booleans are not needed

Reply:
@sadiuk TODO